modules/qna/src/views/FormModal.jsx

```diff
@@ -260,10 +270,8 @@ export default class FormModal extends Component {
           </Modal.Body>
 
           <Modal.Footer className={style.qnaModalFooter}>
-            <Button className={style.qnaModalFooterCancelBtn} onClick={this.closeAndClear}>
-              Cancel
-            </Button>
-            <Button className={style.qnaModalFooterSaveBtn} type="submit">
+            <Button onClick={this.closeAndClear}>Cancel</Button>
+            <Button bsStyle="primary" type="button" onClick={e => this.handleSubmit(e, isEdit)}>
```

Comment:
you don't need to declare an arrow function here you could simply pass `this.handleSubmit` as callback and "compute the isEdit from the props" in the function